見込み管理のE2Eテストが失敗している

```
見込み管理：入力
    お金に関わるフィールドに、フォーカスを外したときに、コンマを追加します
      1) 半角
      2) 全角
  0 passing (9s)
  2 failing
  1) 見込み管理：入力
       お金に関わるフィールドに、フォーカスを外したときに、コンマを追加します
         半角:
      AssertionError: expected '110,363,684,435,583,820' to equal '435,583,829'
      + expected - actual
      -'110,363,684,435,583,820'
      +'435,583,829'

      at Context.eval (webpack:///./cypress/e2e/prospect/input.cy.ts:31:0)

  2) 見込み管理：入力
       お金に関わるフィールドに、フォーカスを外したときに、コンマを追加します
         全角:
      AssertionError: expected '110,363,684,435,583,820' to equal '435,583,829'
      + expected - actual
      -'110,363,684,435,583,820'
      +'435,583,829'
```

--- a/packages/kokoas-e2e/cypress/e2e/prospect/input.cy.ts
+++ b/packages/kokoas-e2e/cypress/e2e/prospect/input.cy.ts
@@ -27,6 +27,7 @@ describe('見込み管理：入力', () => {
           .focus();
 
         cy.get('@contractAmtField')
+          .clear()
           .type(testVal, { delay: 100 });
 
         cy.get('@contractAmtField').blur();

